packages/block-library/src/navigation/edit/navigation-menu-selector.js

```diff
@@ -118,10 +97,10 @@ function NavigationMenuSelector(
 								const label = decodeEntities( menu.name );
 								return (
 									<MenuItem
-										onClick={ handleSelectClassic(
-											onClose,
-											menu
-										) }
+										onClick={ () => {
+											onClose();
```

Comment:
Decided to keep `onClose` here, as converting a classic menu has a saving/loading period, and keeping the menu open while that was happening felt weird.